palace/utils/configfile.cpp

```diff
@@ -1669,6 +1669,10 @@ void LinearSolverData::SetUp(json &solver)
   divfree_max_it = linear->value("DivFreeMaxIts", divfree_max_it);
   gs_orthog_type = linear->value("GSOrthogonalization", gs_orthog_type);
 
+  // Options related to the estimator.
+  estimator_tol = linear->value("EstimatorTol", estimator_tol);
```

Comment:
If we're breaking out this section, `divfree_tol`, `divfree_max_it`, and `gs_orthog_type` should go here too since they aren't explicitly "preconditioner-specific". Maybe just have the comment read: `"Other linear solver options."`